docs/topics/mta-cli-analyze.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="mta-cli-analyze_{context}"]
+= Performing analysis using the command line
+
+`Analyze` allows running source code and binary analysis using `analyzer-lsp`.
+
+To run analysis on application source code, run the following command:
+
+[source,terminal,subs="attributes+"]
+----
+{mta-cli} analyze --input=<path/to/source/code> --output=<path/to/output/dir>
+----
+
+All flags:
+
+[source,terminal,subs="attributes+"]
+----
+Analyze application source code
+
+Usage:
+  {mta-cli} analyze [flags]
+
+Flags:
+      --analyze-known-libraries   analyze known open-source libraries
+  -h, --help                      help for analyze
+  -i, --input string              path to application source code or a binary
+      --json-output               create analysis and dependency output as json
+      --list-sources              list rules for available migration sources
+      --list-targets              list rules for available migration targets
+  -l, --label-selector string     run rules based on specified label selector expression
+      --maven-settings string     path to a custom maven settings file to use
+      --overwrite                 overwrite output directory
+      --skip-static-report        do not generate static report
+  -m, --mode string               analysis mode. Must be one of 'full' or 'source-only' (default "full")
+  -o, --output string             path to the directory for analysis output
+      --rules stringArray         filename or directory containing rule files
+      --skip-static-report        do not generate static report
+  -s, --source string             source technology to consider for analysis. To specify multiple sources, repeat the parameter: --source <source_1> --source <source_2> etc.
+  -t, --target string             target technology to consider for analysis. To specify multiple sources, repeat the parameter: --source <source_1> --source <source_2> etc.
+
+Global Flags:
+      --log-level uint32   log level (default 4)
+      --no-cleanup         do not cleanup temporary resources
+----
+.Usage example
+
+. Get an example application to run analysis on.
+. List available target technologies.
++
+[source,terminal,subs="attributes+"]
+----
+{mta-cli} analyze --list-targets
+----
+. Run an analysis with a specified target technology.
```

Comment:
nit: maybe it would be good to specify that `cloud-readiness` is an example, something like:

> Run an analysis with a specified target technology, ie `cloud-readiness`: